[2025-10-11T15:01:18.223+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-10-11T15:01:18.259+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: read_sqlite_db.read_customers manual__2025-10-11T15:01:14.288967+00:00 [queued]>
[2025-10-11T15:01:18.282+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: read_sqlite_db.read_customers manual__2025-10-11T15:01:14.288967+00:00 [queued]>
[2025-10-11T15:01:18.283+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2025-10-11T15:01:18.347+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): read_customers> on 2025-10-11 15:01:14.288967+00:00
[2025-10-11T15:01:18.352+0000] {standard_task_runner.py:72} INFO - Started process 40 to run task
[2025-10-11T15:01:18.355+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'read_sqlite_db', 'read_customers', 'manual__2025-10-11T15:01:14.288967+00:00', '--job-id', '14', '--raw', '--subdir', 'DAGS_FOLDER/read_sqlite_db.py', '--cfg-path', '/tmp/tmpovgkagn0']
[2025-10-11T15:01:18.357+0000] {standard_task_runner.py:105} INFO - Job 14: Subtask read_customers
[2025-10-11T15:01:18.411+0000] {logging_mixin.py:190} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/settings.py:209 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-10-11T15:01:18.496+0000] {task_command.py:467} INFO - Running <TaskInstance: read_sqlite_db.read_customers manual__2025-10-11T15:01:14.288967+00:00 [running]> on host 5538f3f005ec
[2025-10-11T15:01:18.687+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='read_sqlite_db' AIRFLOW_CTX_TASK_ID='read_customers' AIRFLOW_CTX_EXECUTION_DATE='2025-10-11T15:01:14.288967+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-10-11T15:01:14.288967+00:00'
[2025-10-11T15:01:18.688+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-10-11T15:01:18.736+0000] {logging_mixin.py:190} INFO - âœ… Retrieved data from SQLite:
[2025-10-11T15:01:18.737+0000] {logging_mixin.py:190} INFO - (1, 'Customer 1', 'alice@example.com', '2025-10-11 14:13:49')
[2025-10-11T15:01:18.737+0000] {logging_mixin.py:190} INFO - (2, 'Bob', 'bob@example.com', '2025-10-11 14:13:49')
[2025-10-11T15:01:18.737+0000] {logging_mixin.py:190} INFO - (3, 'Charlie', 'charlie@example.com', '2025-10-11 14:13:49')
[2025-10-11T15:01:18.738+0000] {python.py:240} INFO - Done. Returned value was: [(1, 'Customer 1', 'alice@example.com', '2025-10-11 14:13:49'), (2, 'Bob', 'bob@example.com', '2025-10-11 14:13:49'), (3, 'Charlie', 'charlie@example.com', '2025-10-11 14:13:49')]
[2025-10-11T15:01:18.776+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-10-11T15:01:18.777+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=read_sqlite_db, task_id=read_customers, run_id=manual__2025-10-11T15:01:14.288967+00:00, execution_date=20251011T150114, start_date=20251011T150118, end_date=20251011T150118
[2025-10-11T15:01:18.811+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-10-11T15:01:18.836+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-10-11T15:01:18.839+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
